_topic_map.yml

```diff
@@ -214,6 +214,38 @@ Topics:
     Distros: openshift-enterprise,openshift-dedicated
 
 ---
+Name: Complete Container Security
```

Comment:
I'm wondering if this should just be "Container Security". I'm thinking that having "Complete" would come under the same umbrella of why we don't use "best practices", ie, we can't claim that it's the _best_, much like we shouldn't claim this is _complete_.